sixtyfps_runtime/corelib/model.rs

```diff
@@ -459,12 +469,18 @@ impl<T> Model for ModelHandle<T> {
         self.0.as_ref().map_or(0, |model| model.row_count())
     }
 
-    fn row_data(&self, row: usize) -> Self::Data {
-        self.0.as_ref().unwrap().row_data(row)
+    fn row_data(&self, row: usize) -> Option<Self::Data> {
+        if let Some(model) = self.0.as_ref() {
+            model.row_data(row)
+        } else {
+            None
+        }
```

Comment:
Slightly shorter variant perhaps :-)

```suggestion
        self.0.as_ref().and_then(|model| model.row_data(row))
```